test/embedded/runtime.swift

```diff
@@ -0,0 +1,83 @@
+// RUN: %target-run-simple-swift(-enable-experimental-feature Embedded -parse-as-library -wmo -Xfrontend -disable-objc-interop) | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: optimized_stdlib
+// REQUIRES: VENDOR=apple
+// REQUIRES: OS=macosx
+
+@_silgen_name("putchar")
+func putchar(_: UInt8)
```

Comment:
Just a test, but this is also claiming `putchar` uses Swift calling convention. We'll get away with it in most cases, certainly on Darwin, because it's very similar to the C calling convention. Ideally `@_cdecl` would support doing the same thing `@_silgen_name` does, which would avoid the need for this.